extensions/objects/templates/list-of-sets-of-html-strings-editor.directive.html

```diff
@@ -13,15 +13,22 @@
   p {
     margin-bottom: 10px;
   }
+  .oppia-drag-and-drop-error {
+    position: absolute;
+  }
```

Comment:
Why is this needed?

Reply:
Actually, without this, the modal was getting resized every time the error gets rendered. 

![clunky](https://user-images.githubusercontent.com/35144226/80291612-a99bd000-876c-11ea-85f5-41374e6b25e8.gif)

I know it's minor, but it was annoying.